lib/elixir/lib/module/parallel_checker.ex

```diff
@@ -175,13 +201,21 @@ defmodule Module.ParallelChecker do
     state
   end
 
-  defp spawn_checkers(%{modules: [{map, binary} | modules]} = state) do
+  defp spawn_checkers(%{modules: [{module, _} = verify | modules]} = state) do
     parent = self()
     ets = state.ets
+    send_results_pid = state.send_results
 
     spawn_link(fn ->
-      {binary, warnings} = Module.Checker.verify(map, binary, {parent, ets})
-      send(parent, {__MODULE__, map.module, binary, warnings})
+      case Module.Checker.verify(verify, {parent, ets}) do
```

Comment:
Just a suggestion: you could also make verify always return `{chunk, warnings}`, returning `{nil, []}` for the error case, which would allow you to get rid of this case. The current way also works. :+1: